content/posts/2024-02-26-GSoC.md

```diff
@@ -0,0 +1,74 @@
+  * [Adapting IR Load Semantics to Freeze All or Freeze Only Uninitialized Data](https://summerofcode.withgoogle.com/archive/2023/projects/sc4OauXM) by John McIver, mentored by Nuno Lopes
+  * [Addressing Rust optimization failures in LLVM](https://summerofcode.withgoogle.com/archive/2023/projects/a16FfPnb) by Kohei Asano, mentored by Nikita Popov
+  * [Tutorial development with clang-repl](https://summerofcode.withgoogle.com/archive/2023/projects/qxuEqL8W) by Krishna Narayanan, mentored by Vassil Vassilev
+  * [Fix Handling of Undefined Behavior in NewGVN](https://summerofcode.withgoogle.com/archive/2023/projects/oQUfcx89) by Manuel Brito, mentored by Nuno Lopes
+  * [Map LLVM values to corresponding source-level expressions](https://summerofcode.withgoogle.com/archive/2023/projects/JToY28Mf) by phyBrackets, mentored by Satish Guggilla and Karthik Senthil
+  * [Machine Learning Guided Ordering of Compiler Optimization Passes](https://summerofcode.withgoogle.com/archive/2023/projects/T8rB84Sr) by Puneeth A R, mentored by Tarindu Jayatilaka, Johannes Doerfert, and Mircea Trofin
+  * [Patch based test coverage for quick test feedback](https://summerofcode.withgoogle.com/archive/2023/projects/TMnF4MqC) by ShivamGupta123, mentored by Henrik Olsson
+  * [Re-optimization using JITLink](https://summerofcode.withgoogle.com/archive/2023/projects/4vqXC8Ez) by Sunho Kim, mentored by Vassil Vassilev, Stefan Gränitz, and Lang Hames
+  * [Improvements in Clang Diagnostics](https://summerofcode.withgoogle.com/archive/2023/projects/wxRFR261) by Takuya Shimizu, mentored by Timm Bäder
+  * [Build & Run SingleSource Benchmarks with ClangIR](https://summerofcode.withgoogle.com/archive/2023/projects/Yofn8VfG) by Vinicius Espindola, mentored by Bruno Cardoso Lopes and Nathan Lanza
+  * [Better Performance Models for MLGO Training](https://summerofcode.withgoogle.com/archive/2023/projects/wiTzRdnV) by Viraj Shah, mentored by Mircea Trofin, Aiden Grossman, and Ondrej Sykora
+  * [Enhancing llvm-cov to Generate Hierarchical Coverage Reports](https://summerofcode.withgoogle.com/archive/2023/projects/ghJRTsUJ) by Yuhao Gu, mentored by Petr Hosek and Gulfem Savrun Yeniceri
+  * [Autocompletion in Clang-REPL](https://summerofcode.withgoogle.com/archive/2023/projects/Gxg82Pvd) by Yuquan Fu, mentored by Vassil Vassilev
+
+Some projects also provided detailed end-of-project reports or project diaries that are outstanding on their own:
+ * [Tutorial Development with Clang-Repl](https://blog.llvm.org/posts/2023-10-5-tutorial-development-with-clang-repl/)
+ * [Diagnostic Improvements in Clang 17](ttps://blog.llvm.org/posts/2023-09-19-diagnostic-improvements-in-clang-17/)
+ * [Improving Compile Times](https://dc03.github.io/)
+ * [Addressing Rust optimization failures in LLVM](https://khei4.github.io/gsoc2023/)
+
+ Few others will be added to the list above soon.
```

Comment:
I would probably drop this line. If they aren't included in this post, most likely people won't read a second time.